catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/associations/Associated.java

```diff
@@ -262,20 +283,22 @@ private Filter forChildAssociations(Metacard metacard) {
 
     }
 
-    private Map<String, Metacard> query(Filter filter)
+    private Map<String, Metacard> query(List<String> writableSourceIds, Filter filter)
             throws UnsupportedQueryException, SourceUnavailableException, FederationException {
         if (filter == null) {
             return Collections.emptyMap();
         }
 
-        QueryResponse query = catalogFramework.query(new QueryRequestImpl(new QueryImpl(filter,
+        QueryRequestImpl queryRequest = new QueryRequestImpl(new QueryImpl(filter,
                 1,
-                0,
+                querySize,
```

Comment:
❓ Should we consider waiting for https://github.com/codice/ddf/pull/2080 to get merged and use the new result iterator/pagination utility? That way we wouldn't need to ever worry about `querySize` getting too big or capped.